fe/fe-core/src/test/java/com/starrocks/server/SharedDataStorageVolumeMgrTest.java

```diff
@@ -870,4 +870,28 @@ public void testCreateStorageVolumeWithInvalidParams() throws DdlException {
                     () -> svm.createStorageVolume(svName, "s3", locations, params, Optional.empty(), ""));
         }
     }
+
+    @Test
+    public void testUpgrade() throws IOException, SRMetaBlockException, SRMetaBlockEOFException,
+            DdlException, AlreadyExistsException {
+        StorageVolumeMgr svm = new SharedDataStorageVolumeMgr();
+        svm.createBuiltinStorageVolume();
+        svm.replayBindDbToStorageVolume(null, 1L);
```

Comment:
we should block this call with `null` as parameter.

Reply:
done

Reply:
done